src/utils/networked-waypoint-schema.ts

```diff
@@ -0,0 +1,28 @@
+import { NetworkedWaypoint } from "../bit-components";
+import { defineNetworkSchema } from "./define-network-schema";
+import { deserializerWithMigrations, Migration, NetworkSchema, read, StoredComponent, write } from "./network-schemas";
+import type { EntityID } from "./networking-types";
+
+const runtimeSerde = defineNetworkSchema(NetworkedWaypoint);
+
+const migrations = new Map<number, Migration>();
+
+function apply(_eid: EntityID, { version }: StoredComponent) {
+  if (version !== 1) return false;
+  return true;
+}
+
+export const NetworkedWaypointSchema: NetworkSchema = {
+  componentName: "networked-waypoint",
+  serialize: runtimeSerde.serialize,
+  deserialize: runtimeSerde.deserialize,
+  // TODO: Strange to have to provide serializeForStorage for components
+  // that will not go to storage.
```

Comment:
Perhaps if a schema doesn't provide `serializeForStorage` and `deserializeFromStorage`, we can interpret that as, "don't store data for this component"